### PR TITLE
Custom headers configuration

### PR DESCRIPTION
**Pull request by gzumba**
This makes it easier to add custom headers in your project.

The PR is against 3.0.x and can be safely applied to all branches as it only adds a new configuration key and does not break anything and brings no mandatory changes to your code base.

I also added a few words in the docs about this.

### CODE REVIEW

File: docs/pages/configuration.md [hunk 1]
```diff
@@ -55,6 +55,28 @@ patchlevel_event_sourcing:
...
+
+```yaml
+patchlevel_event_sourcing:
+  headers: '%kernel.project_dir%/src/*/Domain/Header'
+```
+Or use an array to specify multiple paths.
+
+```yaml
+patchlevel_event_sourcing:
+  events:
```
Comment (DanielBadura) [inline docs/pages/configuration.md]:
```suggestion
  headers:
```

**Review by DanielBadura — APPROVED** [review]
besides the copy paste typo LGTM 👍

**Comment by DanielBadura** [comment]
About the target branch: this patch can be applied safely on all branches, but this is tbf a feature from bundle pov.

But i would be fine to put this in 3.0.x, wdyt @DavidBadura ?

**Comment by DavidBadura** [comment]
Thank you! This is definitely a feature of the bundle.

I changed the target. Can you rebase the branch to 3.2.x?

**Review by DavidBadura — APPROVED** [review]

**Comment by DavidBadura** [comment]
Thank you!

**Pull request merged**